Assets/Project/Scripts/Common/Entities/UserSettings.cs

```diff
@@ -64,8 +64,14 @@ static UserSettings()
             }
 
             CurrentLanguage.Subscribe(language => PlayerPrefsUtility.SetObject(Constants.PlayerPrefsKeys.LANGUAGE, language));
-            BGMVolume.Merge(SEVolume)
-                .Subscribe(_ => SoundManager.Instance.ResetVolume());
+            BGMVolume.Subscribe(value => {
+                PlayerPrefs.SetFloat(Constants.PlayerPrefsKeys.BGM_VOLUME, value);
```

Comment:
Treevel 内で他にもダメなところはたくさんあるんですが、とりあえず。
`Save()` を呼ばなくてもアプリ終了時に保存されますが、アプリがクラッシュした際は保存されないようなので `Save()` を読んでおいた方が良いそうです。記録データではなく、`BGM` や `SE` だから最悪消えてもええやろ、ってことなら、この修正をしなくても良いです。

```suggestion
                PlayerPrefs.SetFloat(Constants.PlayerPrefsKeys.BGM_VOLUME, value);
                PlayerPrefs.Save();
```

Reply:
クラッシュして保存されなくてもそんなに影響ないと思うので対応しないとさせてください